roles/common/tasks/main.yml

```diff
@@ -68,6 +68,7 @@
 
 # Include serial console before kernel-tuning to build serial_console_cmdline
 - include: serial-console.yml tty={{ common.serial_console }}
+  when: common.serial_console.enabled
```

Comment:
So looking back at this change, it seems to be the reason that the PR is failing. Could the ```|bool``` filter not be necessary in this instance?